client/src/components/DataFiles/DataFilesModals/DataFilesUploadModalListing/DataFilesUploadModalListingTable.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Table } from 'reactstrap';
+import { LoadingSpinner } from '_common';
+import { findSystemDisplayName } from 'utils/systems';
+import { FileLengthCell } from '../../DataFilesListing/DataFilesListingCells';
+
+const DataFilesUploadStatus = ({ i, removeCallback }) => {
+  const status = useSelector(state => state.files.operationStatus.upload[i]);
+  switch (status) {
+    case 'UPLOADING':
+      return <LoadingSpinner placement="inline" />;
+    case 'SUCCESS':
+      return <span className="badge badge-success">SUCCESS</span>;
+    case 'ERROR':
+      return <span className="badge badge-danger">ERROR</span>;
+    default:
+      return (
+        <button
+          type="button"
+          className="btn btn-link"
+          onClick={() => removeCallback(i)}
+        >
+          Remove
+        </button>
+      );
+  }
+};
+DataFilesUploadStatus.propTypes = {
+  i: PropTypes.string.isRequired,
+  removeCallback: PropTypes.func.isRequired
+};
+
+function DataFilesUploadModalListingTable({ uploadedFiles, setUploadedFiles }) {
+  const params = useSelector(state => state.files.params.FilesListing);
+  const systemList = useSelector(state => state.systems.systemList);
+  const systemDisplayName = findSystemDisplayName(systemList, params.system);
+
+  const removeFile = id => {
+    setUploadedFiles(uploadedFiles.filter(f => f.id !== id));
+  };
+
+  return (
+    <div style={{ marginTop: '10px' }}>
+      <span style={{ fontSize: '20px' }}>
+        Uploading to {systemDisplayName}/{params.path}
+      </span>
+
+      <div>
```

Comment:
I think we need it, because depending of the parameters we need to change DataFilesUploadModalListingTable position, this is happening only in the calling component DataFilesUploadModal. It's there a better way to do it?

Reply:
That makes sense. But to be clear, is [line 50](https://github.com/TACC/Core-Portal/pull/309/files/9f93b1bf2187f48e9844ef1f30e47b13ee0c53e8#diff-f2ab71f6b46acb37665a041d36d7f61a5760aacd0415aa4a514957c634294031R50) needed? It's a `<div>` with no styling, wrapping another `<div>` on [line 51](https://github.com/TACC/Core-Portal/pull/309/files/9f93b1bf2187f48e9844ef1f30e47b13ee0c53e8#diff-f2ab71f6b46acb37665a041d36d7f61a5760aacd0415aa4a514957c634294031R51).